glide.lock

```diff
@@ -1,13 +1,22 @@
-hash: 6b127577a871ff8e66cd84563cd19e9b2c339a81c9e27adc37808dab3c8a1d5a
-updated: 2016-11-14T09:22:05.45573152-08:00
+hash: 69df4410f726670e69ae64ae30dbd2b1c0c5594dda5ca732c15266f852602dfa
```

Comment:
When a developer adds a new dependency to the glide.yaml file then a manual `glide update` should ensure the glide.lock file gets updated as well.  I've never seen it fail to do so.

A real concern I have with glide that I have not figured out yet, is that `glide update` doesn't seem to be possible to scope at all.  It seems to run on ALL your dependencies, not just on the 1 new package you added.  That can be dangerous as it causes a whole lot of churn all at once which can cause unanticipated regressions.